bindings/go/oci/repository/repository.go

```diff
@@ -98,13 +98,13 @@ func buildResolver(client remote.Client, repository *ocirepospecv1.Repository) (
 		return nil, fmt.Errorf("a base url is required")
 	}
 
-	purl, err := runtime.ParseURLAndAllowNoScheme(repository.BaseUrl)
+	baseURL := strings.TrimSuffix(repository.BaseUrl, "/")
+	purl, err := runtime.ParseURLAndAllowNoScheme(baseURL)
```

Comment:
What about fixing it in `ParseURLAndAllowNoScheme`?

Reply:
I don't want to touch that because it's part of the identity matching.

Reply:
Also, I'm not using the end result there. :) I need the right baseURL :)